packages/blockchain-wallet-v4-frontend/src/assets/locales/index.d.ts

```diff
@@ -1958,7 +1958,7 @@ type MessagesType = {
   'scenes.interest.tab.history': 'Transaction History'
   'scenes.interest.verifyid': 'Upgrade Now'
   'scenes.interest.kycunderreview': 'Gold Verification In Review'
-  'scenes.interest.subheader_store': 'Store crypto and watch it grow.'
+  'scenes.interest.subheader_store': 'Deposit crypto and watch it grow. Interest is paid by the end of the day on the 1st of each month.'
```

Comment:
need to update the id on the translations as well otherwise lokalize wont pick it up